docs/development.md

```diff
@@ -28,6 +28,41 @@ To fully develop and debug SDS, you'll want to have the following installed on y
 
 ## Azure development
 
+### ARM Template Deployment
+
+To create a full deployment environment that would match production requirements, ARM templates have been created to stand up a resource group with
+the following resources included:
+
+- Azure Kubernetes Cluster
+- Managed Identites for benchmark, candidate, laboratory, worker and boostrap roles
+- A Development Jump Box (optional)
+- Azure Firewall
+- Private Link configurations and Virtual Networks
+- Azure Storage accounts
+- Azure Container Registry
+
+To deploy the ARM template, first create an SSH key on the machine you intend to execute the ARM template from - it should be named id_rsa (the default name when using [ssh keygen](https://www.tutorialspoint.com/unix_commands/ssh-keygen.htm)).
+
+#### Using local templates (optional)
+
+>If you are developing changes in the ARM templates and wish to use them directly, you first need to make them available over the internet (due to limitations in the Azure CLI). The easiest way to do this is using the [ngrok](https://ngrok.com/download) tool. Once you have installed the ngrok tool, you can expose the ARM template folder by running the command:
```

Comment:
Can we update this to point to using the tools that are baked into the repo? We have an npm script to auto-launch ngrok with the correct settings, and then `deploy.sh` will automatically introspect the running ngrok instance (as long as `--assets` isn't explicitly set) to get the URL & auto-fill the assets URI. Deployment from a fresh clone is:

```shell
# install dependencies
npm install

# Start a local tunnel
npm run dev:azure

# deploy templates
./deploy/arm/deploy.sh -g <resource_group> --dev
```

Reply:
Updated to reflect npm tooling